core/dataflow/src/test/java/edu/uci/ics/texera/dataflow/plangen/LogicalPlanTest.java

```diff
@@ -356,10 +385,10 @@ public void testInvalidLogicalPlan2() throws Exception {
      * KeywordSource --> RegexMatcher --<
      *                                   -> TupleSink2
      */
-    @Test(expected = TexeraException.class)
+    @Test
     public void testInvalidLogicalPlan3() throws Exception {
```

Comment:
change this to `testLogicalPlan4` since such scenario is no longer invalid.

Reply:
`testLogicalPlan4` is a disconnected graph. Should we change it?

Reply:
`testInvalidLogicalPlan4` is the disconnected graph right? I am saying since the function `testInvalidLogicalPlan3` use to be called `invalid` because it describes the scenario of multiple sink operators. But now we allow multiple sink operators so it should be a `testLogicalPlanX` instead of `testInvalidLogicalPlan`. then since we already have `testLogicalPlan1``testLogicalPlan2`and`testLogicalPlan3`, we should be renaming `testInvalidLogicalPlan3` to `testValidLogicalPlan4` right?

Reply:
Yes, you are right.  
I will move this test in right place.